spec/models/review_spec.rb

```diff
@@ -3,12 +3,13 @@
 RSpec.describe Review, type: :model do
   let(:user) { FactoryBot.create(:user) }
   let(:movie) { FactoryBot.create(:movie) }
-  let(:review) { FactoryBot.build(:review) }
+  let(:review) { FactoryBot.build(:review, user: user, movie: movie) }
   let(:review2) { FactoryBot.create(:review, user_id: user.id, movie_id: movie.id) }
   let(:review3) { FactoryBot.build(:review, user_id: user.id, movie_id: movie.id) }
   let(:invalid_review) { FactoryBot.build(:invalid_review) }
 
   it { is_expected.to validate_presence_of(:body) }
+  it { is_expected.to validate_presence_of(:movie) }
 
```

Comment:
And speaking of changes to rspec support gems, `shoulda-matchers` has a different syntax now too: 
```ruby
# from
 it { is_expected.to validate_presence_of(:movie) }

# to
it { should validate_presence_of(:movie) }
```
If what we have is working now, I'm not sure we need to update it at this point int time.